dot/telemetry/telemetry.go

```diff
@@ -76,7 +78,6 @@ func (h *Handler) AddConnections(conns []*genesis.TelemetryEndpoint) {
 		c, _, err := websocket.DefaultDialer.Dial(v.Endpoint, nil)
 		if err != nil {
 			fmt.Printf("Error %v\n", err)
```

Comment:
`continue` 
Don't append a connection that returns an error.

Reply:
Good catch, updated.